doc/simultaneous_trajectory_execution/simultaneous_trajectory_execution_tutorial.rst

```diff
@@ -0,0 +1,47 @@
+Simultaneous Trajectory Execution
+==================================
+
+Introduction
+------------
+MoveIt now allows simultaneous execution of trajectories, as long as, each trajectory uses a different set of controllers. For example, in a dual arm environment, each arm can execute a different set of trajectories without needing to wait for the other arm to finish moving or manually synchronizing the motion of both arm into a single trajectory. Optionally, a collision check is performed right before execution of new trajectories to prevent collisions with active trajectories.
+
+The following GIF shows simple example of simultaneous execution of trajectories through the **Rviz Motion Planning** plugin.
+
+.. only:: html
+
+   .. figure:: images/simultaneous-execution-rviz.gif
+
+This tutorial presents how to use the Simultaneous Trajectory Execution feature through the `Move Group C++ Interface <../move_group_interface/move_group_interface_tutorial.html>`_ but it can be similarly used through the `Move Group Python Interface <../move_group_python_interface/move_group_python_interface_tutorial.html>`_. 
+
+Getting Started
+---------------
+If you haven't already done so, make sure you've completed the steps in `Getting Started <../getting_started/getting_started.html>`_.
+
+Setup
+---------------
+The simultaneous trajectory execution feature can be enabled or disabled through the dynamic reconfigure parameter **/move_group/trajectory_execution/enable_simultaneous_execution**.
+Optionally, an extra layer of collision checking, done right before execution of trajectories, can be enabled through the dynamic reconfigure parameter **/move_group/trajectory_execution/enable_collision_checking**.
```

Comment:
Both parameters are on by default, here it sounds like they are both off by default

Reply:
Right, I change the explanation to make it clear that it is on by default, but it is configurable.